frontend/src/components/Utils/time.js

```diff
@@ -77,9 +77,30 @@ export function getTimestampFromDateString(dateStr) {
 /**
  * Formats a Firestore timestamp into a date string in ISO format.
  *
+ * No tests for this function because there is no logic.
+ *
  * @param {firebase.firestore.Timestamp} timestamp Firestore timestamp object.
  * @return {string} ISO formatted date string: "YYYY-MM-DD or 2020-05-12".
  */
 export function timestampToISOString(timestamp) {
   return timestamp.toDate().toISOString().substring(0,10);
 }
+
+
+/**
+ * Returns the string date range of the trip associated with the Trip document
+ * start and end date timestamps.
+ *
+ * No tests for this function because there is no logic.
+ *
+ * @param {!firebase.firestore.Timestamp} startDateTimestamp Firestore timestamp
+ *     Object corresponding to the trip start date.
+ * @param {!firebase.firestore.Timestamp} endDateTimestamp Firestore timestamp
```

Comment:
I must've missed it if you did it before but what is the exclamation marks for?

Reply:
[Non-null objects for JSDoc](https://github.com/google/closure-compiler/wiki/Types-in-the-Closure-Type-System#the-javascript-type-language)